app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala

```diff
@@ -384,7 +384,8 @@ class SubsidyController @Inject()(
   }
 
   private def getJourneyNext(journey: SubsidyJourney)(implicit request: Request[_]) =
-    if(journey.isAmend()) Future.successful(Redirect(routes.SubsidyController.getCheckAnswers())) else journey.next
+    if(journey.isAmend) Future.successful(Redirect(routes.SubsidyController.getCheckAnswers()))
```

Comment:
We should probably agree on some format and use scalafmt. I don't really mind the format - as long as it's consistent.

Reply:
Yeah again, there's a scala convention about when to use parens which intellij flags, which is why I fixed it.

Reply:
Agree we should aim for a consistent format since it'll reduce noise in PRs.